lib/scss/stanford_events.node.scss

```diff
@@ -0,0 +1,149 @@
+@charset 'UTF-8';
+@import "./stanford_events.config";
+
+.section-editorial-content {
+  @include modular-spacing('padding', 6 null 3 null);
+  @include modular-spacing('margin', null null 5 null);
+
+  color: $su-color-black;
+
+  h1 {
+    @include modular-spacing('margin', null null -1 null);
+    @include modular-typography(5);
+  }
+
+  .su-event-type {
+    @include subheading;
+  }
+
+  .su-event-dek {
+    @include big-paragraph;
+    @include modular-spacing('margin', null null 4 null);
+  }
+
+  .su-event-sponsor {
+    @include subheading;
+  }
+}
+
+.body-region {
+  .node-stanford-event-su-event-type {
+    @include modular-spacing('margin', 5 null);
+
+    h2 {
+      @include big-paragraph;
+      @include modular-spacing('margin', null null -3 null);
+      @include modular-spacing('padding', 0 null null null);
+
+      border-top: 1px solid $su-color-cool-grey-15;
+    }
+  }
+}
+
+.right-region {
+  @include small-paragraph;
+  @include modular-spacing('padding', null null null -1);
+
+  .jumpstart-ui-page-heading {
+    h2 {
+      @include modular-typography(1);
+      @include modular-spacing('margin', null null 0 null);
+    }
+  }
+
+  .node-stanford-event-su-event-location,
+  .node-stanford-event-su-event-alt-loc,
+  .node-stanford-event-su-event-telephone,
+  .node-stanford-event-su-event-audience {
+    h2 {
+      @include small-paragraph;
+
+      font-weight: $su-font-bold;
+    }
+  }
+
+  .node-stanford-event-su-event-date-time {
+    @include modular-spacing('margin', null null 2 null);
+  }
+
+  .su-event-location {
+    @include modular-spacing('margin', null null 2 null);
+
+    p {
+      margin-bottom: 0;
+    }
+  }
+
+  .node-stanford-event-su-event-email {
+    @include modular-spacing('margin', null null 2 null);
+  }
+}
+
+.main-region {
```

Comment:
@jenbreese Is this `main-region` selector needed for specificity or can it be dropped?